distribution/openrct2.d.ts

```diff
@@ -579,6 +579,15 @@ declare global {
         getAllEntities(type: "staff"): Staff[];
         getAllEntities(type: "car"): Car[];
         getAllEntities(type: "litter"): Litter[];
+        getAllEntitiesOnTile(type: EntityType, x: number, y: number): Entity[];
+        /**
+         * @deprecated since version 34, use guest or staff instead.
+         */
+        getAllEntitiesOnTile(type: "peep", x: number, y: number): Peep[];
```

Comment:
Since this is a new function, it doesn't make sense to immediately deprecate it. Just remove the "peep" support.

Reply:
Done: https://github.com/OpenRCT2/OpenRCT2/pull/15901/commits/e8735b161066c57ab4085323925f0e92cfdceaa4